Emergence/Contexts/Main Menu/ShowsOverviewViewController.swift

```diff
@@ -42,13 +41,20 @@ class ShowsOverviewViewController: UICollectionViewController, UICollectionViewD
         requestShowsAtIndex(1)
     }
 
+    override func viewDidAppear(animated: Bool) {
+        super.viewDidAppear(animated)
+
+        guard let nav = navigationController else { return }
+        nav.viewControllers = nav.viewControllers.filter({ $0.isKindOfClass(AuthViewController) == false })
```

Comment:
Good use of `guard` – scratching my brain to figure out a way to abstract this away with the type system.